src/elements/Tag/Tag.stories.tsx

```diff
@@ -0,0 +1,47 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from "react";
+import { Story, Meta } from "@storybook/react";
+import { Tag, TagProps, TagType } from "./Tag";
+
+export default {
+    title: "Elements/Tag",
+    component: Tag,
+    args: {
+        label: "Label",
+    },
+    argTypes: {
+        type: {
+            options: [...Object.keys(TagType)],
+            mappings: [...Object.values(TagType)],
```

Comment:
`Object.keys` and `Object.values` already return arrays
```suggestion
            options: Object.keys(TagType),
            mappings: Object.values(TagType),
```